package.json

```diff
@@ -48,7 +50,10 @@
     "sequelize": "^5.15.0",
     "sequelize-cli": "^5.5.0",
     "swagger-ui-express": "^4.0.7",
-    "underscore": "^1.9.1"
+    "underscore": "^1.9.1",
+    "validator": "^11.1.0",
+    "validator.js": "^2.0.3",
```

Comment:
why `validator.js` and `validatorjs`

Reply:
I wanted to use validator in the first place but I wasn't comfortable with it so I opt for vaidatorjs